app/assets/javascripts/test/backend-snapshot-tests/annotations.e2e.js

```diff
@@ -0,0 +1,137 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"peerDependencies": true}] */
+/* eslint-disable import/first */
+// @flow
+import test from "ava";
+import { resetDatabase } from "../enzyme/e2e-setup";
+import * as api from "admin/admin_rest_api";
+import { APITracingTypeEnum } from "admin/api_flow_types";
+
+test.before("Reset database", async () => {
+  resetDatabase();
+});
+
+test("getAnnotationInformation()", async t => {
+  const annotationId = "570b9ff12a7c0e980056fe8f";
+  const annotation = await api.getAnnotationInformation(annotationId, "skeleton");
+  t.is(annotation.id, annotationId);
+  t.snapshot(annotation, { id: "annotations-getAnnotationInformation" });
+});
+
+test.serial("finishAnnotation() and reOpenAnnotation() for task", async t => {
+  const annotationId = "78135c192faeb34c0081c05d";
+  const finishedAnnotation = await api.finishAnnotation(annotationId, APITracingTypeEnum.Task);
+  t.is(finishedAnnotation.state, "Finished");
+  // $FlowFixMe: Make tracingTime deterministic
+  finishedAnnotation.task.tracingTime = 0;
```

Comment:
I'm confused by these assignments to tracingTime, why would it not be deterministic? Afaik the tracing time will only be modified if there are update actions modifying the tracing, right?

Reply:
I experienced varying numbers in tracingTime which is why I overwrote it here. Not sure why the number increases without updateActions? @fm3 Do you know more here?